packages/core/src/__tests__/__e2e__/multiline-input/MultilineInput.cy.tsx

```diff
@@ -122,4 +122,34 @@ describe("GIVEN an MultilineInput", () => {
           });
       });
   });
+
+  it("should collapse back to fit content when content is reduced", () => {
+    cy.mount(<Default />);
+    cy.findByRole("textbox")
+      .invoke("height")
+      .then((defaultHeight) => {
+        cy.findByRole("textbox").realClick();
+        cy.realPress("Enter");
+        cy.realPress("Enter");
+        cy.realPress("Enter");
+        cy.findByRole("textbox")
+          .invoke("height")
+          .then((newHeight) => {
+            expect(newHeight ?? 0).to.be.greaterThan(defaultHeight ?? 0);
+          });
+      });
+    cy.findByRole("textbox")
+      .invoke("height")
+      .then((defaultHeight) => {
+        cy.findByRole("textbox").realClick();
+        cy.realPress("Escape");
```

Comment:
I'm assuming this shouldn't be Escape?

Reply:
Also this isn't testing the bug you found which is around setting the value prop directly. This would still call onChange and work without the code change below